[2025-03-16T04:38:33.792+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-16T04:38:33.805+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.gold_layer_group.load_dimensions_and_facts manual__2025-03-16T04:38:15.159125+00:00 [queued]>
[2025-03-16T04:38:33.812+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.gold_layer_group.load_dimensions_and_facts manual__2025-03-16T04:38:15.159125+00:00 [queued]>
[2025-03-16T04:38:33.812+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-03-16T04:38:33.823+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): gold_layer_group.load_dimensions_and_facts> on 2025-03-16 04:38:15.159125+00:00
[2025-03-16T04:38:33.830+0000] {standard_task_runner.py:72} INFO - Started process 1832 to run task
[2025-03-16T04:38:33.835+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'gold_layer_group.load_dimensions_and_facts', 'manual__2025-03-16T04:38:15.159125+00:00', '--job-id', '168', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpqx0qk6fh']
[2025-03-16T04:38:33.836+0000] {standard_task_runner.py:105} INFO - Job 168: Subtask gold_layer_group.load_dimensions_and_facts
[2025-03-16T04:38:33.879+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.gold_layer_group.load_dimensions_and_facts manual__2025-03-16T04:38:15.159125+00:00 [running]> on host c891f3a5df31
[2025-03-16T04:38:33.957+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='gold_layer_group.load_dimensions_and_facts' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T04:38:15.159125+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-16T04:38:15.159125+00:00'
[2025-03-16T04:38:33.958+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-16T04:38:33.962+0000] {log.py:232} WARNING - 2025-03-16 04:38:33.961 | INFO     | data_pipeline.gold.load_to_dwh:load_dimensions_and_facts:215 - Loading features into Data Warehouse
[2025-03-16T04:38:33.964+0000] {log.py:232} WARNING - 2025-03-16 04:38:33.964 | INFO     | data_pipeline.gold.load_to_dwh:load_dimensions_and_facts:221 - Available columns: []
[2025-03-16T04:38:33.965+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-16T04:38:33.966+0000] {base.py:84} INFO - Retrieving connection 'postgres_***'
[2025-03-16T04:38:33.969+0000] {sql.py:624} INFO - Running statement: CREATE SCHEMA IF NOT EXISTS feature_store;, parameters: None
[2025-03-16T04:38:33.970+0000] {log.py:232} WARNING - 2025-03-16 04:38:33.970 | INFO     | data_pipeline.gold.load_to_***:create_trip_features:171 - Creating trip features table
[2025-03-16T04:38:33.973+0000] {log.py:232} WARNING - 2025-03-16 04:38:33.972 | ERROR    | data_pipeline.gold.load_to_***:load_dimensions_and_facts:232 - Failed to load trip features: "None of [Index(['trip_id', 'vendor_id', 'pickup_location_id', 'dropoff_location_id',\n       'payment_type', 'rate_code_id', 'event_timestamp', 'trip_distance',\n       'fare_amount', 'tip_amount', 'total_amount', 'passenger_count',\n       'trip_duration_minutes', 'avg_speed_mph', 'is_shared_ride',\n       'is_suspicious', 'created_timestamp'],\n      dtype='object')] are in the [columns]"
[2025-03-16T04:38:33.973+0000] {log.py:232} WARNING - 2025-03-16 04:38:33.973 | INFO     | data_pipeline.gold.load_to_***:create_location_features:196 - Creating location features table
[2025-03-16T04:38:33.973+0000] {log.py:232} WARNING - 2025-03-16 04:38:33.973 | ERROR    | data_pipeline.gold.load_to_***:load_dimensions_and_facts:240 - Failed to load location features: "None of [Index(['location_id', 'latitude', 'longitude', 'event_timestamp',\n       'current_demand', 'available_taxis', 'surge_multiplier',\n       'weather_condition', 'traffic_level', 'created_timestamp'],\n      dtype='object')] are in the [columns]"
[2025-03-16T04:38:33.974+0000] {python.py:240} INFO - Done. Returned value was: {'success': True, 'trip_features_loaded': 0, 'location_features_loaded': 0}
[2025-03-16T04:38:33.999+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-16T04:38:33.999+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=gold_layer_group.load_dimensions_and_facts, run_id=manual__2025-03-16T04:38:15.159125+00:00, execution_date=20250316T043815, start_date=20250316T043833, end_date=20250316T043833
[2025-03-16T04:38:34.051+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-16T04:38:34.063+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-16T04:38:34.065+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
